src/main/java/uk/gov/companieshouse/api/accounts/util/logging/LogBuilderImpl.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.companieshouse.api.accounts.util.logging;
+
+import static uk.gov.companieshouse.api.accounts.CompanyAccountsApplication.APPLICATION_NAME_SPACE;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+public class LogBuilderImpl implements LogBuilder {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAME_SPACE);
+
+  private String requestId;
+
+  private String userId;
+
+  private String transactionId;
+
+  private String accountsId;
+
+  public LogBuilderImpl(String requestId, String userId, String transactionId, String accountsId) {
+    this.requestId = requestId;
+    this.userId = userId;
+    this.transactionId = transactionId;
+    this.accountsId = accountsId;
+  }
+
+  public void logStartOfRequestProcessing(String message) {
+    Map<String, Object> logData = new HashMap();
+    addValues(logData);
+    putValue(logData, "message", message);
+    putValue(logData, "identity", userId);
+    LOGGER.infoContext(requestId, "", logData);
```

Comment:
A builder shouldn't be sending the log messages. A build is responsible for building complex objects.